app/Services/VocabularyServiceProvider.js

```diff
@@ -2,41 +2,41 @@ const { VocabularyCard, Translation } = require('../../database');
 const { Drawer } = require('../../database');
 
 // create language package
-async function createVocabularyCard({ languagePackageId, groupId }, name, userId, activate) {
+async function createVocabularyCard({ languagePackageId, groupId }, name, description, userId, activate, res) {
```

Comment:
could we move the description and active to one object?

Reply:
Won't work that well as description is from body and activate from query parameter.
Only thing I could do is wrapping it to an object before giving it to the function, but I don't know if this will improve the code structure.

Reply:
Oh sorry, I overread that.